config/locales/grape.he.yml

```diff
@@ -0,0 +1,5 @@
+he:
+  grape:
+    errors:
+      messages:
+        length_max: 'is expected to have length less than or equal to %{max}'
```

Comment:
Grape uses i18n now, but by default only supports english. I've added file for Hebrew (only one message for now)